.rubocop.yml

```diff
@@ -27,6 +28,33 @@ AllCops:
 
   StyleGuideCopsOnly: false
 
+Capybara:
+  Enabled: true
+
+Capybara/ClickLinkOrButtonStyle:
+  Enabled: true
+
```

Comment:
We should have all `rubocop-capybara` cops listed here. Several are missing.

Can you also list these in alphabetical order, please?

Reply:
is there a command line to list them all?

Reply:
I don't know. When I set up Rubocop for a work project, I just copied the rules from the docs using some JS in a browser console. When I checked the `rubocop-capybara` docs, it was only a handful of rules missing. I haven't set up `rubocop-capybara` before, otherwise I would have shared the full list of cops like for `rubocop-performance`.

Reply:
I've used all the cops listed at https://docs.rubocop.org/rubocop-capybara/cops.html

Reply:
Awesome thanks! These all look good.